osclib/freeze_command.py

```diff
@@ -197,6 +203,21 @@ def prj_meta_for_bootstrap_copy(self):
             a = ET.SubElement(r, 'arch')
             a.text = 'ppc64le'
 
+        if with_product:
+            r = ET.SubElement(root, 'repository', {'name': 'product', 'linkedbuild': 'all'})
+            ET.SubElement(r, 'path', {'project': self.prj, 'repository': 'standard'})
+
+            if self.prj.startswith('SUSE:'):
```

Comment:
do you think it is worth to encapsulate this logic:

```
        r = ET.SubElement(root, 'repository', {'name': 'images', 'linkedbuild': 'all'})
        ET.SubElement(r, 'path', {'project': self.prj, 'repository': 'standard'})

        if self.prj.startswith('SUSE:'):
            a = ET.SubElement(r, 'arch')
            a.text = 'local'
        a = ET.SubElement(r, 'arch')
        a.text = 'x86_64'

        if 'ppc64le' in self.api.cstaging_archs:
            a = ET.SubElement(r, 'arch')
            a.text = 'ppc64le'
```

as it appears to be the same for both the images and product repositories?

Reply:
Fixed this in v3